server/etcdserver/apply/apply.go

```diff
@@ -146,8 +146,8 @@ func newApplierV3Backend(
 		txnModeWriteWithSharedBuffer: txnModeWriteWithSharedBuffer}
 }
 
-func (a *applierV3backend) Apply(ctx context.Context, r *pb.InternalRaftRequest, applyFunc applyFunc) *Result {
-	return applyFunc(ctx, r)
+func (a *applierV3backend) Apply(r *pb.InternalRaftRequest, applyFunc applyFunc) *Result {
+	return applyFunc(context.TODO(), r)
```

Comment:
Suggest to add similar comment as in https://github.com/etcd-io/etcd/pull/18674 right above this line to make this PR completely independent. In the future, when anyone continue to completely get rid of the context, then we can remove the comment by then.

Reply:
I have all commit ready. Just going one by one.

Reply:
https://github.com/etcd-io/etcd/pull/18676 is next